components/Picture.js

```diff
@@ -0,0 +1,13 @@
+import React, { PropTypes } from 'react';
+
+const Picture = (props) => (
+  <div className="picture">
+    <img src={props.url} alt={props.url} />
```

Comment:
`alt` should be passed in as a string and be descriptive. Using the url is pretty bad.